db.json

```diff
@@ -2111,7 +2163,8 @@
     "extensions": ["air"]
   },
   "application/vnd.adobe.flash.movie": {
-    "source": "iana"
+    "source": "iana",
+    "extensions": ["swf","spl"]
   },
```

Comment:
https://www.iana.org/assignments/media-types/application/vnd.adobe.flash.movie mentions the commonly used .swf , but not .spl

Reply:
Added in https://gitlab.freedesktop.org/xdg/shared-mime-info/-/commit/298543c7e449165b46a85d71209faf42de508217 - .sql is added with the media type of Futuresplash

https://www.webdesignmuseum.org/software/futuresplash-animator-in-1996 lists "FutureSplash Player SPL (Export)" under image formats, not animation formats, so I suspect that they are two different formats.

https://en.wikipedia.org/wiki/SWF lists .spl as a "Related file formats and extension"

https://github.com/ruffle-rs/ruffle/issues/18260 and https://github.com/ruffle-rs/ruffle/blob/295bd1062cfa281beeb4371f44b3cfb53c12171b/web/packages/core/src/swf-utils.ts#L24 says .spl is the same as .swf

First few pages of https://github.com/search?q=path%3A*.spl&type=code are not flash.

I am not 100% sure, but this seems to be ok.